src/Imdb/Title.php

```diff
@@ -785,7 +785,7 @@ public function tagline()
         if ($this->main_tagline == "") {
             $taglines = $this->taglines();
 
-            $this->main_tagline = $taglines[0] ?? '';
+            $this->main_tagline = $taglines[0] ? $taglines[0] : '';
```

Comment:
I think this would be closer to what the null coalesce is doing. Presumably if $taglines was [] $taglines[0] would give an array offset not found error.
```php
!isset($taglines[0]) ? $taglines[0] : ''
```

Reply:
You are right. I just changed the PHP style. I didn't fix any possible bug or PHP warnings.

Reply:
Ah, I got your point. You are right. I changed the code.